plans/main.fmf

```diff
@@ -0,0 +1,41 @@
+summary: Prepare CI DNF Stack
+adjust:
+    - when: distro == fedora-37
+      environment:
+          BASE: "fedora:37"
+    - when: distro == fedora-38
+      environment:
+          BASE: "fedora:38"
+    - when: distro == fedora-39
+      environment:
+          BASE: "fedora:39"
+    - when: distro == fedora-rawhide
+      environment:
+          BASE: "fedora:rawhide"
```

Comment:
This adjustments are a bit unfortunate - we will need to adjust them every time a new Fedora is branched. Would it be possible to use original distro value and do something like this?
```
...container-test build --base $(echo "$BASE" | tr '-' ':') ...
```

Reply:
for fedora no problem but we'll need to adjust stuff for centos, rhel etc in the future

Reply:
@m-blaha I tweaked the `--base` option but we will need to be smarter if we'll want to add Centos tests too

Reply:
Thank you.